ci-operator/config/openshift/agent-installer-utils/openshift-agent-installer-utils-release-4.15.yaml

```diff
@@ -37,6 +37,19 @@ releases:
       include_built_images: true
       name: "4.15"
       namespace: ocp
+  s390x-initial:
```

Comment:
Not sure what's the recommended way to handle ci-operator configs for arch different than x86, but in any case I guess that the openshift/installer folder may be a better target rather than agent-installer-utils

Reply:
@andfasano 

In consultation with @deepsm007 from CI we have added this config for s390x to target latest builds for the same in our workflow. 

As per our previous discussion in January along with @v78singh , we came to a conclusion that it's okay if we keep s390x workflows in a new directory under agent-intsaller-utils itself. Please confirm this.

Reply:
Steps registry folder to store ibmz workflow it's ok.
My concern it's about placing s390x _jobs_ inside `.../config/openshift/agent-installer-utils`, which is a minor one used by ABI - and moreover rarely updated. The main repo for ABI is `.../config/openshift/installer`, which seems to me a more suitable config repo for such jobs.

Reply:
Removed the s390x images from presubmits as we will be running periodics now.

Reply:
The comment was for all the types of jobs, not just the presubmits. The `agent-installer-utils` component repo folder it's not the right place for a ibmz periodic job testing the whole installation. As in general I'm not opposed in having periodics outside the [relase folder](https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/#periodics-outside-of-openshiftrelease), there could be better alternatives for placing the periodic - for example `openshift/multiarch` seems a suitable config folder ?

Reply:
@andfasano , okay I believe there are other e2e workflows which does the e2e whole cluster installation on x86 in the same agent-installer-utils directory.

@tvardema , Please suggest where to place the e2e agent based installation on IBMZ workflow periodic job

Reply:
@phani2898 that's the common pattern to verify the current repo (in this case `openshift/agent-installer-utils`) patches, because for each PR (commit) the CI creates an ephemeral payload assembled with the latest available release version + a fresh component image (usually, to be promoted) built with the content of the current patch.. So, the e2e _presubmits  jobs_ are defined to verify any new specific `agent-installer-utils` before merging it.

Reply:
okay sure @andfasano , @jaypoulz / @tvardema Please suggest where to place the e2e agent based installation on IBMZ workflow periodic job.